headers/augment_headers.py

```diff
@@ -128,17 +129,19 @@ def input_header_file(self):
     def output_header_file(self):
         return self.slist.header_file + self.out_extension
 
-    def commit_header_file(self):
-        self.formatter.format_file(self.output_header_file())
+    def commit_header_file(self, format = True):
```

Comment:
`format` needs a type annotation:

```suggestion
    def commit_header_file(self, format: bool = True):
```